src/formats.js

```diff
@@ -5,7 +5,8 @@ const formats = {
   TD2: 'TD2',
   TD3: 'TD3',
   SWISS_DRIVING_LICENSE: 'SWISS_DRIVING_LICENSE',
-  FRENCH_NATIONAL_ID: 'FRENCH_NATIONAL_ID'
+  FRENCH_NATIONAL_ID: 'FRENCH_NATIONAL_ID',
+  FRENCH_DRIVING_LICENSE: 'FRENCH_DRIVING_LICENSE'
```

Comment:
Could we call it EU_DRIVING_LICENSE or the french one has non-standard parts?

Reply:
There is no standard in EU driving license. As I know, French, Swiss and Netherlands driving licenses are all has unique format.